### DISCUSSION TITLE
Living Design Document

### DISCUSSION THREAD

**Discussion by SirGiraffey** [fishfight/punchy#209] [General]
Hey, I made a design document for Punchy and I'd love it if you could check it out and give feedback on its contents!
[link](https://hackmd.io/cCwOIxkUQTapNeiEUZI0sg?view)

**Comment by erlend-sh** [comment]
Great work! We will probably incorporate this into our newly established [Wiki](https://github.com/fishfolks/punchy/wiki) soon.

The Credits section needs to be updated further, but we might want to replace it with a standardized CREDITS.md file instead. Will ponder that a bit.
